src/Engine/Utils.java

```diff
@@ -93,17 +93,17 @@ public static void highlightLocations(GameMap map, ArrayList<XYCoord> locations)
   }
 
   /** Alias for {@link #findPossibleDestinations(XYCoord, Unit, GameMap) findPossibleDestinations()} **/
-  public static ArrayList<XYCoord> findPossibleDestinations(Unit unit, GameMap gameMap)
+  public static ArrayList<XYCoord> findPossibleDestinations(Unit unit, GameMap gameMap, boolean includeOccupied)
   {
-    return findPossibleDestinations(new XYCoord(unit.x, unit.y), unit, gameMap);
+    return findPossibleDestinations(new XYCoord(unit.x, unit.y), unit, gameMap, includeOccupied);
```

Comment:
Just as an aside, `includeOccupiedSpaces`, though slightly more verbose, is definitely more descriptive.